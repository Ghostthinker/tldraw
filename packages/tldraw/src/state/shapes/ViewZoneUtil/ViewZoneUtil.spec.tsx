import {ViewZone} from '..'

describe('Viewzone shape', () => {
  it('Creates a shape', () => {
    expect(ViewZone.create({ id: 'viewzone' })).toMatchSnapshot('viewzone')
  })
})
