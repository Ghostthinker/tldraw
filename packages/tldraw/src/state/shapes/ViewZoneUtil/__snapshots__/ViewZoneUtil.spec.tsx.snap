// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viewzone shape Creates a shape: viewzone 1`] = `
Object {
  "childIndex": 1,
  "id": "viewzone",
  "label": "",
  "labelPoint": Array [
    0.5,
    0.5,
  ],
  "name": "ViewZone",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": true,
    "scale": 1,
    "size": "small",
  },
  "type": "viewzone",
}
`;
